easy-install.py

```diff
@@ -0,0 +1,239 @@
+                check=True,
+            )
+            print(
+                f"{CYLW}Please go through the Development Documentation: https://github.com/frappe/frappe_docker/tree/main/development to fully complete the setup.{CEND}"
+            )
+        except Exception as e:
+            print(f"{CRED}Setting Up Development Environment Failed\n{CEND}", e)
+    else:
+        install_docker()
+        clone_frappe_docker_repo()
+        setup_dev_instance(project)  # Recursion on goes brrrr
+
+
+def install_docker():
+    if which("docker") is None:
+        print(f"{CGRN}Docker is not installed, Installing Docker...{CEND}")
+        if platform.system() == "Darwin" or platform.system() == "Windows":
+            print(
+                f"""{CRED}
+                This script doesn't install Docker on {"Mac" if platform.system()=="Darwin" else "Windows"}.
+
+                Please go through the Docker Installation docs for your system and run this script again{CEND}"""
+            )
+            exit(1)
+        try:
+            ps = subprocess.run(
+                ["curl", "-fsSL", "https://get.docker.com"],
+                capture_output=True,
+                check=True,
+            )
+            subprocess.run(["/bin/bash"], input=ps.stdout, capture_output=True)
+            subprocess.run(
+                ["sudo", "usermod", "-aG", "docker", str(os.getenv("USER"))], check=True
+            )
+            print(f"{CYLW}Waiting Docker to start{CEND}")
+            time.sleep(10)
+            subprocess.run(
+                ["sudo", "systemctl", "restart", "docker.service"], check=True
+            )
+        except Exception as e:
+            print(f"{CRED}Failed to Install Docker{CYLW}\n", e)
+            print(
+                f"\n\n {CYLW}Try Installing Docker Manually and re-run this script again{CEND}\n\n"
+            )
+            exit(1)
+    else:
+        return
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Install Frappe with Docker")
+    parser.add_argument(
+        "-p", "--prod", help="Setup Production System", action="store_true"
+    )
+    parser.add_argument(
+        "-d", "--dev", help="Setup Development System", action="store_true"
+    )
+    parser.add_argument(
+        "-s",
+        "--sitename",
+        help="The Site Name for your production site",
+        default="site1.local",
+    )
+    parser.add_argument(
+        "-n",
+        "--project",
+        help="Project Name",
+        default="frappe",
+    )
+    parser.add_argument(
+        "--email",
+        help="Add email for the SSL.",
+        required="--prod" in sys.argv
+    )
+    args = parser.parse_args()
+    if args.dev:
+        print(f"\n{CGRN}Setting Up Development Instance{CEND}\n")
+        setup_dev_instance(args.project)
+    elif args.prod:
+        print(f"\n{CGRN}Setting Up Production Instance{CEND}\n")
+        if "example.com" in args.email:
```

Comment:
why this specific check? Many dummy emails out there.

Reply:
traefik will throw error if it is example.com for letsencrypt/certbot notification email.